docs/reference/commandline/images.md

```diff
@@ -155,3 +155,24 @@ Ready for use by `docker rmi ...`, like:
 NOTE: Docker will warn you if any containers exist that are using these untagged images.
 
 
+##### Labelled images
+
+The `label` filter allows to filter image based on the presence of the label and its value.
+
+    $ docker images --filter "label=com.example.version"
+
+    REPOSITORY          TAG                 IMAGE ID            CREATED              VIRTUAL SIZE
+    match-me-1          latest              eeae25ada2aa        About a minute ago   188.3 MB
+    match-me-2          latest              eeae25ada2aa        About a minute ago   188.3 MB
+
+This will display all images that have a label `com.example.version` no matter what value it has.
+
+    $ docker images --filter "label=com.example.version=1.0"
+    REPOSITORY          TAG                 IMAGE ID            CREATED              VIRTUAL SIZE
+    match-me            latest              eeae25ada2aa        About a minute ago   188.3 MB
+
+This will display all images that have a label `com.example.version` with the value `1.0`. If nothing matches, the list will be empty.
```

Comment:
The following filter matches images with the `com.example.version` label with the `1.0` value.  In this example returns an empty set because no matches were found.